### DISCUSSION TITLE
Intent 6 analysis

### DISCUSSION THREAD

**Discussion by Kevsy** [camaraproject/EdgeCloud#177] [General]
### Developer intents: Provisioning intents

| Intent 6 | 5SED |5MEE |EXRA|EXRC| EXCA | EXSM|  GMEC |
| -------- |------| ----|----| ---|------|-----|-------|
| 6. "I can ask the operator to inform me if the optimal MEC for my application and a specific terminal changes, taking into account mobility events, connectivity, shortest network path, cost, network load, MEC platform load etc."  | No           |No    | ?          | ?     | ?            | ?     | No   |

| 5MEE           | Details       |Comments|
| --------------|-------------| -----------|
| (to complete) | (to complete) | (to complete) |

| EXRC           | Details       |Comments|
| -------------  |:-------------:| ------:|
| (to complete) | (to complete) | (to complete) |

(to add EXRA, EXCA,  EXSM,  GMEC as appropriate)

**Comment by ThomasEXR** [comment]
Our platform does have a mechanism to exactly cater for that intent (Edge Events - see https://developers.edgexr.org/sdks/edge-events-overview/ for reference) but that is built as streaming API not as a REST API (and it's between app and operator platform, not between developer and operator platform) thus we haven't considered this yet for CAMARA. Therefore it will be a "No" for the EdgeXR contributions.

**Comment by maheshc01** [comment]
the version of the API spec contributed does not support user subscriptions and notifications but 5GFF has an updated API spec which supports this and the plan is to contribute it to CAMARA as a generic module which can be use across multiple APIs

**Comment by gunjald** [comment]
Currently the Application management API submitted does not provide the capabilities to provide change of optimal MEC notifications to applications considering the various events in context of the UE e.g. mobility events, QoS deterioration, network congestion etc
